### ISSUE TITLE
Make a Dedicated Space for the Contributors to the Website

### ISSUE THREAD

**Issue by velitasali** [trebleshot/trebleshot.github.io#6]
Mentioning those who contributed to the project's website on a visible space could drive more people to the project.

# Expected look

* Shows the details for the contributors. It may include a Twitter handle,  GitHub profile, name, nickname, personal website.
* It should be visible in about or contributors page. 

# How it should work

* It should use YAML, **_config.yml** to be more specific.
* It can be added by the contributors themselves.

**Comment by mubashir-rehman** [comment]
https://training.galaxyproject.org/training-material/hall-of-fame This might help.
It is a bit messy cuz I can't find the files where these should be located. Looking forward to implement it before Hacktober ends.

**Comment by velitasali** [comment]
No problem at all and yes, this can help. 

_site.data_ seems to be the source (https://github.com/galaxyproject/training-material/search?q=contributors), but it is not in the _config.yml file. They may be lazy loading it at some point.

**Comment by mubashir-rehman** [comment]
The data is in -config.yml but I can't find it. However, I found another solution that automates everything but currently I don't have knowledge for adding social links. https://showndarya.github.io/Hacktoberfest/Contributors_Page_Source/ this uses github api and the code is very simple. It might cause conflict with base theme but it is an option. The code is here https://github.com/Showndarya/Hacktoberfest/tree/master/Contributors_Page_Source

**Comment by velitasali** [comment]
Yes, that seems to be using the jQuery for ajax call. We already have jQuery for Bootstrap, so adding this shouldn't cause too much trouble. The call happens here: https://github.com/Showndarya/Hacktoberfest/blob/master/Contributors_Page_Source/index.js

**Comment by mubashir-rehman** [comment]
#15 
Not an ideal page but it works.
